### ISSUE TITLE
Show toolbar style from SE 3.x

### ISSUE THREAD

**Issue by niksedk** [SubtitleEdit/subtitleedit#7323]
If you want the toolbar to look like SE 3.x, then go to Options - Settings - Appearance - Graphics button theme: Legacy

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/17dcc648-c485-4d84-b063-8a52c363c79e)

![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/95bae229-aa60-4cb3-a668-b9e3b4bb7100)

**Comment by MbuguaDavid** [comment]
There's a problem with the player... It's the old one.
![Problem --- Old Windows Media Player Icons](https://github.com/SubtitleEdit/subtitleedit/assets/92556454/a9115064-221a-4051-9f79-2154d631de5e)

**Comment by niksedk** [comment]
> There's a problem with the player... It's the old one.


Thx David, will be fixed in next update...

**Comment by oep42** [comment]
I do appreciate the option to choose the Legacy Theme. When choosing this, I would expect to see all buttons of the legacy theme, not only the buttons at the top of the screen and of the player, but also the buttons below the waveform. In SE 4.0.0 NEXT beta 27, after choosing the Legacy Theme, there do appear, below the waveform, buttons that are not "legacy" at all, and without a visible cue of the setting of the Center button. I would appreciate it, if the buttons below the waveform would be real legacy, when choosing the Legacy Theme. 

With the size of all items set to 125% in the Windows settings, the percentage indication below the waveform looks rather useless in SE 4.0.0 NEXT beta 27. 

See the images below of the old user interface and the current legacy user interface.

![SE - Old Interface](https://github.com/SubtitleEdit/subtitleedit/assets/84353335/f36d54b7-25a8-48f4-99b5-2b5f9332eb7e)
![SE - Legacy Interface](https://github.com/SubtitleEdit/subtitleedit/assets/84353335/cab5f2a3-97e1-46d8-95d3-872478208b37)

**Comment by niksedk** [comment]
> I do appreciate the option to choose the Legacy Theme. When choosing this, I would expect to see all buttons of the legacy theme, not only the buttons at the top of the screen and of the player, but also the buttons below the waveform. In SE 4.0.0 NEXT beta 27, after choosing the Legacy Theme, there do appear, below the waveform, buttons that are not "legacy" at all, and without a visible cue of the setting of the Center button. I would appreciate it, if the buttons below the waveform would be real legacy, when choosing the Legacy Theme.

If you hoover the mouse, you will get a hint:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/2385892b-9daa-4fcb-b083-d0092c9952ca)

The old images were poor for the dark theme, so they were updated... you liked the old "center" icon better?

I've tried to make the zoom percentage box wider: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.0/SubtitleEditBeta.zip

**Comment by oep42** [comment]
The zoom percentage box is wider now, and works well. Thank you. 

In my opinion, the new images look rather ugly, and do not 'work' as well as the old ones. (I do not use the dark theme, so I did not take the dark theme into consideration.) I can understand when people say that the new design of the images is clearer than the old ones. Anyway, I guess I will get used to the new design of 'legacy images', even though I didn't expect an update of those images when they are supposed to be legacy.

(Just before reading your comment, I removed "and without a visible cue of the setting of the Center button" from my earlier comment, because this cue is visible in the Legacy Theme. Sorry about this.)
